Android/app/src/main/res/values-en/strings.xml

```diff
@@ -1,57 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<!-- This is the app's name, translators please do not translate and put in the
-same text as English version. Developers can consider putting in their own
-translation, this string must be changed before publishing -->
+	<!-- This is the app's name, translators please do not translate and put in the same text as English version. Developers can consider putting in their own translation, this string must be changed before publishing -->
```

Comment:
Hi @ar-arvind, why did you change the comments 🤔 

But all in all the pull looks good to me 👍

Reply:
Hi @MarvinJWendt, I just reformatted the file to display the comments in single line. Thanks!